### PR TITLE
feat: Uid file v3

### PR DESCRIPTION
**Pull request by PhilippeWeidmann**

### CODE REVIEW

File: kDriveCore/Data/Models/File.swift [hunk 1]
```diff
@@ -730,13 +731,20 @@ public final class File: Object, Codable {
         return ProxyFile(driveId: driveId, id: id)
     }
 
+    public static func uid(driveId: Int, fileId: Int) -> String {
```
Comment (adrien-coye) [inline kDriveCore/Data/Models/File.swift:734]: To truly be a UID, shouldn't we use the user ID also ?

**Review by adrien-coye — COMMENTED** [review]
All good. Just one question.

File: kDriveCore/Data/Models/File.swift (hunk 1 above)
Comment (PhilippeWeidmann) [reply kDriveCore/Data/Models/File.swift:734]: As you guessed, `driveId` is unique so no need to go further with `userId`.

**Review by adrien-coye — APPROVED** [review]
Thanks for the feedback 🙌

**Pull request merged**